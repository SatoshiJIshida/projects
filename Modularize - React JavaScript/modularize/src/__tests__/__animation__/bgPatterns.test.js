import React from "react";
import { shallow } from "enzyme";
import {
  Connections,
  Movements,
  Movements2,
} from "../../components/animation/bgPatterns";

/**
 * Library Jest taken from source https://jestjs.io/
 * Library Enzyme taken from source https://www.npmjs.com/package/enzyme
 */

describe("Background Screen Pattern Animations", () => {
  it("should be defined", () => {
    expect(Connections).toBeDefined();
    expect(Movements).toBeDefined();
    expect(Movements2).toBeDefined();
  });

  it("Pattern1 should render correctly", () => {
    const anime1 = shallow(<Connections name="anime-test" />);
    expect(anime1).toMatchSnapshot();
  });

  it("Pattern2 should render correctly", () => {
    const anime2 = shallow(<Movements name="anime-test" />);
    expect(anime2).toMatchSnapshot();
  });

  it("Pattern3 should render correctly", () => {
    const anime3 = shallow(<Movements2 name="anime-test" />);
    expect(anime3).toMatchSnapshot();
  });
});
