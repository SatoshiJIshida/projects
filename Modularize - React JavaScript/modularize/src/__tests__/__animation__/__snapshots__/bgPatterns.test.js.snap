// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Background Screen Pattern Animations Pattern1 should render correctly 1`] = `ShallowWrapper {}`;

exports[`Background Screen Pattern Animations Pattern2 should render correctly 1`] = `ShallowWrapper {}`;

exports[`Background Screen Pattern Animations Pattern3 should render correctly 1`] = `ShallowWrapper {}`;
